util/src/main/i18n/templates/moderation.properties

```diff
@@ -175,3 +175,4 @@ vanish.chat.deny = You can only use the admin chat while vanished
 
 vanish.hotbar = You are currently vanished
 
+vanish.login.domain = You logged in with {0} so we auto-vanished you
```

Comment:
No need for special message, the hot bar is fine enough.

Reply:
Ok! I’ll make this change right away